sources/packages/web/src/views/aest/student/CASSupplierInformation.vue

```diff
@@ -151,9 +162,24 @@ export default defineComponent({
           "Unexpected error while loading CAS supplier information.",
         );
       }
+      retryButtonDisabled.value =
+        casSupplierInfo.value.items[0]?.supplierStatus !==
+        SupplierStatus.PendingSupplierVerification;
     };
     watchEffect(() => loadCASSuppliers(props.studentId));
 
+    const retryCASSupplier = async () => {
+      try {
+        await CASSupplierService.shared.retryCASSupplier(props.studentId);
+        snackBar.success("CAS supplier information has been retried.");
```

Comment:
The message seems misleading because can lead the user to understand that a new attempt was made.
I would recommend the following. Feel free to check with biz.
```
A new CAS pending verification record was created. A new attempt will be made to get information from CAS.
```

Reply:
Checked with Biz and changed the AC